core/tests/browse_tests.py

```diff
@@ -0,0 +1,57 @@
+import os
+
+from django.test import TestCase
+from django.db import connection
+from django.conf import settings
+
+from chronam.core.views.browse import get_page_text
+from chronam.core.utils.utils import _get_tip
+from chronam.core.batch_loader import BatchLoader, Batch
+from chronam.core.models import Title, LanguageText
+
+
+class BrowseTests(TestCase):
```

Comment:
Why is this called BrowseTests? Perhaps put a class comment describing what this tests?

Reply:
Yes, I will add a class comment there